packages/ai/src/LLMStep.ts

```diff
@@ -167,7 +186,7 @@ export class LLMStepInstance<const Shape extends StepShape = StepShape> {
     value: Outputs<Shape>[K] | Outputs<Shape>[K]["value"]
   ): void {
     if (key in this.outputs) {
-      this.criticalError(`Output ${key} is already set`);
+      this.fail("MINOR", `Output ${key} is already set`);
```

Comment:
This error is should be critical; in most cases, it's a bug in step implementation.

Reply:
Makes sense. Wasn't very sure about that. 

That said, later on, I imagine some "CRITICAL" errors could lead to it restarting or something.